pkg/logs/metrics/metrics.go

```diff
@@ -0,0 +1,38 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package metrics
+
+import (
+	"expvar"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/status"
+)
+
+var (
```

Comment:
Some logs can be dropped at `decoding`, I think it could be nice to have either the number of logs collected or the number of logs dropped.

Reply:
I'm not sure what would be the best place to hook in for this, so I added a TODO for another PR.
I also added LogsDecoded for the number of Decoded logs (they can be dropped before the processing based on custom rules).

Reply:
thanks